alpha/declcfg/declcfg.go

```diff
@@ -93,17 +100,68 @@ func (m Meta) MarshalJSON() ([]byte, error) {
 }
 
 func (m *Meta) UnmarshalJSON(blob []byte) error {
+	blob = bytereplacer.New(`\u003c`, "<", `\u003e`, ">", `\u0026`, "&").Replace(blob)
```

Comment:
I moved the bytes replacement here, which meant I could drop the extra unmarshal into `json.RawMessage`. Between the allocation improvement of a direct bytes replacer and the removal of the other unmarshal, we shave off ~7% of the runtime of `opm render` when I run it against the operatorhub catalog.

Not too shabby!

In a follow-up, I might take a look at an alternate mode of `action.Render` (maybe a new function alongside `Run()`?), which lets the caller use a walk function. That would save a bunch on memory and might shave off more time as well.